### ISSUE TITLE
Vector nc file missing "distance to bed" variable

### ISSUE THREAD

**Issue by scmcgill0** [USGS-CMG/stglib#211]
Can we have the variable for "range/dist from probe" and/or "range/dist from samp vol" added to the nc file for the vector?

**Comment by dnowacki-usgs** [comment]
Yes, this is possible. Were there variable names for this used in the Matlab scripts?

**Comment by scmcgill0** [comment]
Yes, the names used were: "vrange" ('Volume Range to Boundary') and "brange" ('Sensor Range to Boundary')
